src/main/steps/application-within-proceedings/upload-your-application/content.ts

```diff
@@ -0,0 +1,98 @@
+import { AWPApplicationType, YesOrNo } from '../../../app/case/definition';
+import { TranslationFn } from '../../../app/controller/GetController';
+import { FormContent } from '../../../app/form/Form';
+import { isFieldFilledIn } from '../../../app/form/validation';
+import { getCasePartyType } from '../../../steps/prl-cases/dashboard/utils';
+import { getApplicationDetails } from '../utils';
+
+export const en = (applicationType?: AWPApplicationType): object => ({
+  title: 'Upload your application',
```

Comment:
ideally en & cy shd not be functions. please handle dynamic contents in generatecontent method.

Reply:
en and cy no longer functions and using generateContent method to handle dynamic content now